pkg/kubelet/pod/pod_manager.go

```diff
@@ -237,6 +236,17 @@ func (pm *basicManager) GetUIDTranslations() (podToMirror, mirrorToPod map[types
 
 	podToMirror = make(map[types.UID]types.UID, len(pm.translationByUID))
 	mirrorToPod = make(map[types.UID]types.UID, len(pm.translationByUID))
+	// Insert empty translation mapping for all static pods.
```

Comment:
Can you update this with more comment based on our offline discussion? Without the detail reason & logic behind, I am concerning the maintainability.